packages/mermaid/src/diagrams/class/classDiagram.ts

```diff
@@ -1,20 +1,20 @@
 import { DiagramDefinition } from '../../diagram-api/types.js';
 // @ts-ignore: JISON doesn't support types
 import parser from './parser/classDiagram.jison';
-import db from './classDb.js';
+import classDb from './classDb.js';
 import styles from './styles.js';
 import renderer from './classRenderer.js';
 
 export const diagram: DiagramDefinition = {
   parser,
-  db,
+  db: classDb,
   renderer,
   styles,
   init: (cnf) => {
     if (!cnf.class) {
       cnf.class = {};
     }
     cnf.class.arrowMarkerAbsolute = cnf.arrowMarkerAbsolute;
-    db.clear();
+    classDb.clear();
```

Comment:
nit: This deviates from the convention. `parser`, `styles`, `renderer`, don't have `class` prefix. Which is not really required in this file, as it's specific for classDiagram.